File: Python/1020.py
a = int(input())

ano = int(a / 365)
mes = int((a - (ano * 365)) // 30)
dias = int((a - ((ano * 365) + (mes * 30))))

print(ano, "ano(s)")
print(mes, "mes(es)")
print(dias, "dia(s)")

File: C++/1020.cpp
#include <iostream>
#include <iomanip>
#include <string>
#include <array>
#include <cmath>

using namespace std;

int main(){
    int idade, ano, mes, dia, resto;
    cin >> idade;
    ano = (idade / 365) ;
    resto = idade % 365;
    mes = resto / 30;
    resto = resto % 30;
    dia = resto;

    cout << ano << " ano(s)" << endl;
    cout << mes << " mes(es)" << endl;
    cout << dia << " dia(s)" << endl;
    
}